5 - Git e Github/Tarefa/README.md

```diff
@@ -0,0 +1,23 @@
+# Tarefa
+
+## Seção 1 - Git
+
+Responda as questões a seguir:
+
+1. Qual é a diferença entre Git Merge e Git Rebase?
+
+2. Explique pelo menos uma maneira de resolver um conflito.
```

Comment:
A gente falou sobre isso ou e para o pessoal pesquisar?

Reply:
1. Tem nos slides, mas para uma resposta mais completa o pessoal teria que pesquisar.

2. Falei em aula e tem nos slides

Reply:
Fizeram essa pergunta no início da aula seguinte, tá lá a resposta. Acho uma boa estar aqui também, quem viu a aula vai saber. :)